CMakeLists.txt

```diff
@@ -236,7 +236,6 @@ SET(MAIKO_HDRS
     inc/arraydefs.h
     inc/array.h
     inc/bb.h
-    inc/bbtsubdefs.h
     inc/bbtsub.h
```

Comment:
How about removing bbtsub.h and leaving bbtsubdefs.h (to match the other xxxdefs.h) which also has the #ifdef guard in it ?

Reply:
I'll update this to do that.

Reply:
Updated.